README.md

```diff
@@ -36,19 +31,29 @@ Updating:
 
 If there is migrations that need user input (like renaming a field), they need to be run manually!
 
-Logs / Shell access:
+## Logs / Shell access
 
 - there is some shortcuts in the makefile
 
-Development:
+## Development
 
 - Install dependencies listed in 'Deployment'
 - run 'make build'
 - run 'make dev-up'
 
     NOTE: DEBUG must be set to True in config.env when running development builds
 
-History: 
+## History
 - Development history of the project before 2023.10.26
   - Code: https://gerrit.opnfv.org/gerrit/gitweb?p=laas.git;a=summary
   - Reviews: https://gerrit.opnfv.org/gerrit/q/project:laas
+
+
+## License & Copyright 
+
+Copyright (c) 2016 Max Breitenfeldt and others
```

Comment:
The copyright is different here then in `LICENSE.md`. I'm not really sure how this stuff works legally. Do they need to match?